index.html

```diff
@@ -0,0 +1,265 @@
+        </div>
+      </div>
+    </section>
+    <!-- About Section -->
+    <section class="about-page" id="about-page">
+      <div class="about-container">
+        <div class="about-content">
+          <h2 class="about-title">About Myself</h2>
+          <p class="about-para">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <h3 class="about-contact">LET'S CONNECT</h3>
+          <ul class="social-icons">
+            <li class="icons">
+              <a href="#"><img src="images/Vector (15).png" alt="Twitter" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Linkedin icon.png" alt="LinkedIn"
+              /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (11).png" alt="Medium" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="images/Vector (12).png" alt="GitHub" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="images/Vector (13).png" alt="AngelList"
+              /></a>
+            </li>
+          </ul>
+          <button class="about-btn">Get my resume</button>
+        </div>
+        <div class="about-lang">
+          <ul>
+            <li class="languages">
+                <div class="lang-title">
+                  <h2>Languages</h2>
+                  <img  src="images/Union.svg" alt="arrow" />
+                </div>
+                <ul>
+                  <li class="langs">
+                    <img src="images/Ellipse 1.png" alt="JavaScript-Logo" />
+                    <h4 class="langs-text">JavaSript</h4>
+                  </li>
+                  <li class="langs">
+                    <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
+                    <h4 class="langs-text">HTML</h4>
+                  </li>
+                  <li class="langs">
+                    <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
+                    <h4 class="langs-text">CSS</h4>
+                  </li>
+                </ul>
+            </li>
+            <li>
+                  <div class="framework">
+                    <h2 class="frame-title">Frameworks</h2>
+                    <img src="images/Union.svg" alt="arrow" />   
+                  </div>
+                  <ul class="content skills-list">
+                    <li>React & Redux</li>
+                    <li>Ruby on Rails</li>
+                    <li>Tailwind CSS</li>
+                    <li>Bootstrap</li>
+                    <li>Jest</li>
+                    <li>RSpec</li>
+                  </ul>
+            </li>
+            <li>
```

Comment:
After working on the necessary changes as requested by the previous reviewer, kindly ensure to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🤓 👍

Your design
![image](https://github.com/siddghosh108/Portfolio_Mobile_Finish/assets/78251927/ecdf34e2-e6a3-4a19-8afa-052418b751ab)

Figma
![image](https://github.com/siddghosh108/Portfolio_Mobile_Finish/assets/78251927/813b4cbf-29f6-4215-a1b8-733b54aff582)

Reply:
Now i have changed the script exactly the way it should display just like the figma design keeping the <ul> and <li> structures the way my previous reviewerasked. Please see below

<ul>
            <li class="languages">
                <div class="lang-title">
                  <h2 class="lang-title">Languages</h2>
                  <img  src="images/Union.svg" alt="arrow" />
                </div>
                <ul class="lang">
                  <li class="langs">
                    <img src="images/Ellipse 1.png" alt="JavaScript-Logo" />
                    <h4 class="langs-text">JavaSript</h4>
                  </li>
                  <li class="langs">
                    <img src="images/Ellipse 2.png" alt="HTML5-Logo" />
                    <h4 class="langs-text">HTML</h4>
                  </li>
                  <li class="langs">
                    <img src="images/Ellipse 3.png" alt="CSS3-Logo" />
                    <h4 class="langs-text">CSS</h4>
                  </li>
                </ul>
            </li>
            <li>
                  <div class="framework">
                    <h2 class="frame-title">Frameworks</h2>
                    <img src="images/Union2.svg" alt="arrow" />   
                  </div>
                  <hr class="new1" />
            </li>
            <li>
                  <div class="skill">
                    <h2 class="skill-title">Skills</h2> 
                    <img  src="images/Union2.svg" alt="arrow" />
                  </div>
                  <hr class="new1" />
            </li>
          </ul>

Reply:
![image](https://github.com/siddghosh108/Portfolio_Mobile_Finish/assets/131663359/46c5dd1a-6694-4a37-a915-b49b6fd742db)